components/bl-gallery-component/README.md

```diff
@@ -0,0 +1,107 @@
+# Gallery
+
+Gallery is a component of Backendless UI-Builder designer. It allows you to conveniently view images and stay on the page. 
+
+The component based on external [lightbox2](https://lokeshdhakar.com/projects/lightbox2/) library post by [Lokesh Dhakar](https://github.com/lokesh).
+
+## Usage
+
+ Add component on your page and add codeless logic which loading images.
+ Use ImagesUrl handler to load images in component.    
+ ImagesUrl type is an array of objects, where every object should have field "URL" with image URL and optionally field 'title' with text for every image.
+ All images have the same height and stretch according to the sides, but it is possible to set the height of each image separately. To do this, add the 'height' value to the object.
+ Value must contain px, %, etc...   
+
+    ImagesUrl:[{url:'http..', title:'Picture..', height: '200px' },..]
+Also, you can avoid any logic by writing in settings 'Images Data' links with separating comma ','. 
+
+    htttp://image1, http://image2
+But if you already have logic, this method will not work.
+
+### Options
```

Comment:
Could we try to do markup for options using tables with columns: name, default value, description?